src/pages/post-feed/usersDummyData.ts

```diff
@@ -0,0 +1,56 @@
+export interface User {
+  id: number;
+  username: string;
+  userRank: string;
+  userPhoto: string;
+  text: string;
+  photo: string;
+}
+
+export interface UserType {
+  id: number;
+  userPhoto: string;
+}
```

Comment:
why there is `User` and `UserType`?

Reply:
here I tried to `User` but when I import the object error appear that said `I should use all the keys`  while I need `id` and `userPhoto` wherefore I created UserType.

Reply:
this is confusing, it should either be one type
or if you want to have a minimal type of something you should use something [called `Pick`](https://www.typescriptlang.org/docs/handbook/utility-types.html#picktype-keys)

```ts
export interface UserType {
  id: number;
  username: string;
  userRank: string;
  userPhoto: string;
}

export type MinimalUserType = Pick<UserType, 'id' | 'userPhoto'>
```